distribution/src/main/assembly/dist.xml

```diff
@@ -274,6 +274,33 @@
                 </dependencySets>
             </binaries>
         </moduleSet>
+        <moduleSet>
+            <useAllReactorProjects>true</useAllReactorProjects>
+            <includes>
+                <include>org.hibernate.search:hibernate-search-mapper-orm-coordination-database-polling</include>
```

Comment:
Please also add a dependency in the `pom.xml`. See: https://github.com/hibernate/hibernate-search/blob/f91d624b69ed18c540b4278402a8d3ab4ec72b96/distribution/pom.xml#L30-L38

It will spare us some trouble when using `mvn -am -pl ...`.

And maybe look for the "jsr352" string elsewhere in the distribution project, too. I don't know if there are other tricks.